### PR TITLE
refactor(app): Wire up Error Recovery flows on desktop

### PR DESCRIPTION
**Pull request by TamarZanzouri**
# Overview

closes https://opentrons.atlassian.net/browse/EXEC-552.
@mjhuff 's implementation :-) I just fixed the test and put up the PR. 

# Test Plan

- make sure the error recovery flow matches Figma designs and logic on desktop app.

# Risk assessment

low.

### CODE REVIEW

File: app-shell-odd/src/config/index.ts [hunk 1]
```diff
@@ -122,9 +122,9 @@ export function getOverrides(path?: string): unknown {
 
 export function getConfig<P extends keyof Config>(path: P): Config[P]
 export function getConfig(): Config
-export function getConfig(path?: string): any {
-  const result = path == null ? null : store().get(path)
-  const over = getOverrides(path)
+export function getConfig(path?: any): any {
+  const result = store().get(path)
+  const over = getOverrides(path as string | undefined)
```
Comment (shlokamin) [inline app-shell-odd/src/config/index.ts]: how come we're changing the app shell's config getter?

Comment (TamarZanzouri) [reply app-shell-odd/src/config/index.ts]: @mjhuff can probably speak more about this? maybe @sfoster1 is aware as well?

Comment (shlokamin) [reply app-shell-odd/src/config/index.ts]: it's gone now that you merged edge into the branch

**Review by sfoster1 — APPROVED** [review]
This looks good, just need to get tests/checks passing

**Pull request merged**